ui/components/multichain/import-nfts-modal/import-nfts-modal.js

```diff
@@ -129,6 +131,10 @@ export const ImportNftsModal = ({ onClose }) => {
   };
 
   const validateAndSetAddress = (val) => {
+    setNftAddressValidationError(null);
+    if (!isValidHexAddress(val)) {
+      setNftAddressValidationError(t('invalidAddress'));
+    }
```

Comment:
is there any unit test for this function ? if so , you can add the if case and test it

Reply:
pushed! 🙏